tests/acceptance/features/bootstrap/PublicWebDavContext.php

```diff
@@ -288,18 +288,24 @@ public function downloadPublicFileInsideAFolderWithRange(string $path, string $r
 	 * @param string $password
 	 * @param string $range ignored when empty
 	 * @param string $publicWebDAVAPIVersion
+	 * @param bool $downloadForShareNG
 	 *
 	 * @return ResponseInterface
 	 */
 	public function downloadFileFromPublicFolder(
 		string $path,
 		string $password,
 		string $range,
-		string $publicWebDAVAPIVersion = "old"
+		string $publicWebDAVAPIVersion = "old",
+		bool $downloadForShareNG = false
```

Comment:
can we utilize `publicWebDAVAPIVersion`?
something like `publicWebDAVAPIVersion="shareNg"`
```suggestion
		string $publicWebDAVAPIVersion = "old"
```